### PR TITLE
Add new pipeline to test triggering basic e2e from prow

### PR DESCRIPTION
**Pull request by mboukhalfa**

### CODE REVIEW

**Comment by adilGhaffarDev** [comment]
we are adding a new pipeline script anyhow, can we use dynamic workers here while we are at it?

**Comment by mboukhalfa** [comment]
> dding a new pipeline script anyhow, can we use dynamic workers here while we are at it?
This is just testing the changes with basic e2e once we get this working we can re draw a design in term of the dynamic workers and the pipeline separation

File: jenkins/jobs/prow_integration_tests.pipeline
Comment (Sunnatillo) [inline jenkins/jobs/prow_integration_tests.pipeline:1]: Why the name is prow integration tests? I suggest renaming it basic_e2e_integration_tests.pipeline

**Review by Sunnatillo — COMMENTED** [review]
LGTM, one nit

File: jenkins/jobs/prow_integration_tests.pipeline
Comment (mboukhalfa) [reply jenkins/jobs/prow_integration_tests.pipeline:1]: it is not specific for basic now I just used the same one existing `integration_tests.test ` with changes needed to be triggered from prow

Comment (mboukhalfa) [reply jenkins/jobs/prow_integration_tests.pipeline:1]: What do you suggest ?

Comment (Sunnatillo) [reply jenkins/jobs/prow_integration_tests.pipeline:1]: We had a discussion on a call. This will be changed later, for now we test it first

**Pull request merged**